dts/bindings/mipi-dbi/nxp,lcdic.yaml

```diff
@@ -23,3 +23,26 @@ properties:
     description: |
       Swap bytes while transferring on LCDIC. When set, the LCDIC will send
       the most significant byte first when using multibyte pixel formats.
+
```

Comment:
Do you want to remove "Only SPI mode is currently supported."(L:6)?

Reply:
Yes, we should. I'll update this PR, thanks for catching that